digdag-cli/src/main/java/io/digdag/cli/Command.java

```diff
@@ -54,6 +65,24 @@ protected Command(PrintStream out, PrintStream err)
         this.err = err;
     }
 
+    protected void setupParameters()
```

Comment:
I think it doesn't make sense to have this method at Command because local mode commands don't show tabular records. How about moving this to `ClientCommand` class?